src/render.rs

```diff
@@ -31,7 +31,8 @@ pub fn emit_commandline(app_instance: &AppInstance, overlay_file: &str) -> Vec<S
         "kubecfg",
         "show",
         "--alpha",
-        image,
+        "--reorder=server",
+        &format!("oci://{}", image),
```

Comment:
Without this `oci://` prefix, `kubecfg` doesn't know what to do with the `spec.package.image`. So I believe this fix is correct.